Update README.md (#1467)
LawnchairLauncher/lawnicons: .github/README.md

[![Build debug APK](https://github.com/LawnchairLauncher/lawnicons/actions/workflows/build_debug_apk.yml/badge.svg)](https://github.com/LawnchairLauncher/lawnicons/actions/workflows/build_debug_apk.yml)
[![Build release APK](https://github.com/LawnchairLauncher/lawnicons/actions/workflows/build_release_apk.yml/badge.svg)](https://github.com/LawnchairLauncher/lawnicons/actions/workflows/build_release_apk.yml)
[![Discord](https://img.shields.io/discord/803299970169700402?label=server&logo=discord)](https://discord.gg/lawnchair-803299970169700402)
[![License is Apache 2](https://img.shields.io/github/license/LawnchairLauncher/lawnicons)](./LICENSE)

**Lawnicons** is an icon pack developed by the Lawnchair team.
Originally an addon for Lawnchair 12 Alpha 5 and above to implement themed icons, it can now be used on many launchers. Some launchers may also support Lawnicons for themed icons.

Lawnicons is best used on [Lawnchair 12.1](https://github.com/LawnchairLauncher/lawnchair). You can enable themed icons on Lawnchair by going to `Home Settings > General > Icon Style > Themed Icons`, choosing "Home Screen" or "Home Screen and App Drawer".

> This project is mostly supported by our community. If you wish to contribute, check out the Contributing section below.

Reviewer: ```suggestion
This project is mostly supported by our community. If you wish to contribute, check out the [Contributing](#contributing) section below.
```

Author: Adds unneeded visual clutter

## Download
See the [Releases section](https://github.com/LawnchairLauncher/lawnicons/releases) to download the latest stable build of Lawnicons.

For the development versions with new icons, go to [the nightly.link](https://nightly.link/LawnchairLauncher/lawnicons/workflows/build_debug_apk/develop/Debug%20APK).

## Contributing

Please see the [contributing guide](CONTRIBUTING.md) for information on adding icons or contributing code.

> If you know how to create proper icons already, you can use [icontool.py](/.github/icontool_guide.md) to speed up the process of adding icons. Also see the [Figma contributing guide](https://www.figma.com/community/file/1227718471680779613).

Reviewer: The use of "proper" seems a little aggressive or just unnecessary. Maybe it can be removed or something like this?

```suggestion
If you're already familiar with the guidelines for creating icons, you can use [icontool.py](/.github/icontool_guide.md) to speed up the process of adding icons. Also see the [Figma contributing guide](https://www.figma.com/community/file/1227718471680779613).
```

Author: It just means you should only be using icontool if you know how to make icons

Author: Yes, icontool is a tool that takes an svg and adds it to appfilter
You shouldn't be using it with random svgs though

Reviewer: Making an icon and adding an icon are two different things. Sounds like icontool creates an icon

Author: So should I replace "create" with "make" or what

> If you know how to create proper icons already, you can use [icontool.py](/.github/icontool_guide.md) to speed up the process of adding icons. Also see the [Figma contributing guide](https://www.figma.com/community/file/1227718471680779613).

Reviewer 2: ```suggestion
If you're already familiar with the guidelines for creating icons and you created an icon for lawnicons, you can use [icontool.py](/.github/icontool_guide.md) to speed up the process of adding your icons. Otherwise our [Figma contributing guide](https://www.figma.com/community/file/1227718471680779613) is a great start.
```
I think this would be the clearest

## Requesting icons

To request new icons, please use the [icon request form](https://forms.gle/Fx8vZAiWdW1Tyjo57).

> You can also view the [raw requests](https://docs.google.com/spreadsheets/d/1h3eiJnG2nEdR1DbvemaF1lYthHkzYbXvVFPP0TEEt5k/edit?usp=sharing) or a [summary of the requests](https://docs.google.com/spreadsheets/d/10_o0DjhAGfP8ToDDNQ8M7eCZKtbzXLhIvxuF1XZEql0/edit).

Reviewer: What's the difference between the two spreadsheets?

Reviewer: Ahh I see now. In that case raw requests can get removed? The summary already has everything the raw has

Author: Only if nobody loses track of the original spreadsheet, as the second one is just importing cells from the original one
Should I?

For the development versions with new icons, go to [the nightly.link](https://nightly.link/LawnchairLauncher/lawnicons/workflows/build_debug_apk/develop/Debug%20APK).

## Contributing

Please see the [contributing guide](CONTRIBUTING.md) for information on adding icons or contributing code.

For users familiar with the guidelines, check out [how to use icontool.py](/.github/icontool_guide.md) to speedup the process of adding icons.
If you know how to create proper icons already, you can use [icontool.py](/.github/icontool_guide.md) to speed up the process of adding icons. Also see the [Figma contributing guide](https://www.figma.com/community/file/1227718471680779613).

## Requesting icons

To request new icons, please use the [icon request form](https://forms.gle/Fx8vZAiWdW1Tyjo57).

If you want to see the requested icons (for contributing purposes), you can view the form responses in our 
[publicly available Google Sheets](https://docs.google.com/spreadsheets/d/1h3eiJnG2nEdR1DbvemaF1lYthHkzYbXvVFPP0TEEt5k/edit?usp=sharing) file.

**Note that we currently don't show whether an icon request has been completed or not.**

## License

> Lawnicons is licensed under the [Apache License 2.0](https://www.apache.org/licenses/LICENSE-2.0). See [LICENSE](LICENSE) for additional information.

Reviewer 2: This shouldn't be removed. While there is a License File, it is still nice to see the license at a glance.
